EpiAware/docs/src/showcase/replications/mishra-2020/index.jl

```diff
@@ -52,6 +54,12 @@ It is common to conceptualise the generative process of public health data, e.g
 `EpiAware` is built using the [`DynamicPPL`](https://github.com/TuringLang/DynamicPPL.jl) probabilistic programming domain-specific language, which is part of the [`Turing`](https://turinglang.org/dev/docs/using-turing/guide/) PPL. The structural concept behind `EpiAware` is that each module of an epidemiological model is a self-contained `Turing` [`Model`](https://turinglang.org/DynamicPPL.jl/stable/api/#DynamicPPL.Model-Tuple{}); that is each module is an object that can be conditioned on observable data and sampled from. A complete `EpiAware` model is the composition of these objects using the [`@submodel`](https://turinglang.org/DynamicPPL.jl/stable/api/#DynamicPPL.@submodel) macro.
 "
 
+# ╔═╡ 27d73202-a93e-4471-ab50-d59345304a0b
```

Comment:
wouldn't this flow more naturally if the dependencies were below/in this section vs above all the text?

Reply:
They are, or at least are when I render locally which seems to reflect CI.

I'm not sure why the `.jl` file is saved this way round (this might be a legacy of me trying out different order). But the actual display order is found under `# ╔═╡ Cell order:`

This is a case where the `Pluto` notebooks being nearly human readable compared to (say) `ipynb` is maybe a bit confusing?